tutorials/20_REST_API.ipynb

```diff
@@ -0,0 +1,538 @@
+    "service docker start"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "2. Install Haystack.\n",
+    "\n",
+    "Install the latest version of Haystack from the main branch and all its dependencies. REST API and `xpdf` are also required.  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash \n",
+    "\n",
+    "pip install --upgrade pip\n",
+    "pip install 'farm-haystack[all]'\n",
+    "pip install -e rest_api/\n",
+    "\n",
+    "brew install xpdf # required for PDFToTextConverter node"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "3. Clone Haystack repository.\n",
+    "\n",
+    "Haystack provides a `docker-compose.yml` file that defines a container for Elasticsearch. Clone the Haystack repository to be able to run the `docker-compose.yml` file locally. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "\n",
+    "git clone https://github.com/deepset-ai/haystack.git"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "4. Update the `docker-compose.yml` file.\n",
+    "\n",
+    "The current `docker-compose.yml` file has basic settings and it also provides an image for Elasticsearch container which needs to change. For that, go to `docker-compose.yml` and replace `image: \"deepset/elasticsearch-countries-and-capitals\"` with `image: \"docker.elastic.co/elasticsearch/elasticsearch:7.9.2\"`. The new image is going to provide an empty Elasticsearch instance instead of an instance with some indexed articles about countries and capital cities. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "```yaml\n",
+    "services:\n",
+    "  ...\n",
+    "  elasticsearch:\n",
+    "    image: \"docker.elastic.co/elasticsearch/elasticsearch:7.9.2\"\n",
+    "    ports:\n",
+    "      - 9200:9200\n",
+    "    restart: on-failure\n",
+    "    environment:\n",
+    "      - discovery.type=single-node\n",
+    "      - \"ES_JAVA_OPTS=-Xms1024m -Xmx1024m\"\n",
+    "    healthcheck:\n",
+    "        test: curl --fail http://localhost:9200/_cat/health || exit 1\n",
+    "        interval: 10s\n",
+    "        timeout: 1s\n",
+    "        retries: 10\n",
+    "  ...\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "5. Launch Elasticsearch.\n",
+    "\n",
+    "Go to the directory where `docker-compose.yml` is and start the Elasticsearch container. You might have realised that there are settings for haystack and ui containers provided in the `docker-compose.yml`. They are not necessary for in this tutorial. Run `docker-compose up` with the `elasticsearch` argument so that only the Elasticsearch container launches."
```

Comment:
```suggestion
    "Go to the directory where `docker-compose.yml` is and start the Elasticsearch container. Run `docker-compose up` with the `elasticsearch` argument so that only the Elasticsearch container launches."
```

They don't need these settings now so I'd say it's better not to mention them at al otherwise, it's confusing.